src/interceptors/create-bundle-information-error.interceptor.ts

```diff
@@ -0,0 +1,20 @@
+import { CallHandler, ExecutionContext, Injectable, NestInterceptor } from '@nestjs/common';
+import { map, Observable } from 'rxjs';
+
+@Injectable()
+export class CreateBundleInformationErrorInterceptor implements NestInterceptor {
```

Comment:
Do you think the name is clear enough? I'm wondering if we adjust it slightly to `CreateBundleInformationWarningErrorsInterceptor` or even just `WarningErrorsHeaderInterceptor`?